### PR TITLE
[DOCS] 8.0.1 Release Notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses #1565.

Previews:
- [Updated index page](https://security-docs_1608.docs-preview.app.elstc.co/guide/en/security/master/release-notes.html)
- [Release Notes](https://security-docs_1608.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.0.0.html#release-notes-8.0.1)

### CODE REVIEW

File: docs/release-notes/8.0.asciidoc [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+[discrete]
+[[release-notes-8.0.1]]
+=== 8.0.1
+
+[discrete]
+[[bug-fixes-8.0.1]]
+==== Bug fixes and enhancements
+* Fixes a bug that prevented the `kibana.alert.uuid` field from being populated in EQL sequence shell alerts ({pull}125890[#125890]).
+* Applies updated field aliases to mappings in legacy indices ({pull}125888[#125888]).
+* Updates pre-built detection rules ({pull}125316[#125316]).
```
Comment (joepeeples) [inline docs/release-notes/8.0.asciidoc:13]:
```suggestion
* Updates prebuilt detection rules ({pull}125316[#125316]).
```

**Review by joepeeples — APPROVED** [review]
One tiny edit, LGTM!

File: docs/release-notes/8.0.asciidoc [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+=== 8.0.1
+
+[discrete]
+[[bug-fixes-8.0.1]]
+==== Bug fixes and enhancements
+* Fixes a bug that prevented the `kibana.alert.uuid` field from being populated in EQL sequence shell alerts ({pull}125890[#125890]).
+* Applies updated field aliases to mappings in legacy indices ({pull}125888[#125888]).
+* Updates prebuilt detection rules ({pull}125316[#125316]).
+* Truncates long rule exception descriptions when viewing exception items in rule details ({pull}125145[#125145]).
+* Fixes a bug that caused the import process to fail if an exception list contained exception items with comments ({pull}124909[#124909]).
```
Comment (benironside) [inline docs/release-notes/8.0.asciidoc:15]:
```suggestion
* Fixes a bug that caused the import process to fail if an exception list contained an exception item with comments ({pull}124909[#124909]).
```

**Review by benironside — APPROVED** [review]
Looks good, I left one small suggestion.

**Review by jmikell821 — APPROVED** [review]
LGTM - thanks.

**Pull request merged**